UnitTests/SqlGeneration/SqlStatementTextGeneratorTest.cs

```diff
@@ -428,22 +459,26 @@ public void Build_WithSetOperationCombinedStatement ()
         SetOperationCombinedStatements = { setOperationCombinedStatement }
       });
 
-      _stageMock.Expect (
-          mock => mock.GenerateTextForSelectExpression (_commandBuilder, sqlStatement.SelectProjection))
-          .WhenCalled (mi => ((SqlCommandBuilder) mi.Arguments[0]).Append ("[t].[ID],[t].[Name],[t].[City]"));
-      _stageMock.Expect (mock => mock.GenerateTextForFromTable (_commandBuilder, sqlStatement.SqlTables[0], true))
-          .WhenCalled (mi => ((SqlCommandBuilder) mi.Arguments[0]).Append ("[Table] AS [t]"));
-      _stageMock.Expect (mock => mock.GenerateTextForSqlStatement (_commandBuilder, sqlStatement.SetOperationCombinedStatements[0].SqlStatement))
-          .WhenCalled (mi => ((SqlCommandBuilder) mi.Arguments[0]).Append ("SELECT FOO FROM BAR"));
-      _stageMock.Replay();
+      _stageMock
+          .Setup (mock => mock.GenerateTextForSelectExpression (_commandBuilder, sqlStatement.SelectProjection))
+          .Callback ((ISqlCommandBuilder sqlCommandBuilder, Expression _) => sqlCommandBuilder.Append ("[t].[ID],[t].[Name],[t].[City]"))
+          .Verifiable();
+      _stageMock
+          .Setup (mock => mock.GenerateTextForFromTable (_commandBuilder, sqlStatement.SqlTables[0], true))
+          .Callback ((ISqlCommandBuilder sqlCommandBuilder, SqlTable _1, bool _2) => sqlCommandBuilder.Append ("[Table] AS [t]"))
+          .Verifiable();
+      _stageMock
+          .Setup (mock => mock.GenerateTextForSqlStatement (_commandBuilder, sqlStatement.SetOperationCombinedStatements[0].SqlStatement))
+          .Callback ((ISqlCommandBuilder sqlCommandBuilder, SqlStatement _) => sqlCommandBuilder.Append ("SELECT FOO FROM BAR"))
+          .Verifiable();
```

Comment:
I don't know why we did this this way way back when, but those calls to Verifiable() are so useless, considering we're asserting the commandbuilder's text. Oh well...